packages/kbn-shared-ux-components/src/page_template/page_template.test.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { shallow, render } from 'enzyme';
+import { KibanaPageTemplate } from './page_template';
+import { KibanaPageTemplateSolutionNavProps } from './solution_nav';
+import { NoDataPageProps } from './no_data_page';
+
+const items: KibanaPageTemplateSolutionNavProps['items'] = [
+  {
+    name: 'Ingest',
+    id: '1',
+    items: [
+      {
+        name: 'Ingest Node Pipelines',
+        id: '1.1',
+      },
+      {
+        name: 'Logstash Pipelines',
+        id: '1.2',
+      },
+      {
+        name: 'Beats Central Management',
+        id: '1.3',
+      },
+    ],
+  },
+  {
+    name: 'Data',
+    id: '2',
+    items: [
+      {
+        name: 'Index Management',
+        id: '2.1',
+      },
+      {
+        name: 'Index Lifecycle Policies',
+        id: '2.2',
+      },
+      {
+        name: 'Snapshot and Restore',
+        id: '2.3',
+      },
+    ],
+  },
+];
+
+const solutionNav = {
+  name: 'Kibana',
+  icon: 'logoKibana',
+  items,
+};
+
+const noDataConfig: NoDataPageProps = {
+  solution: 'Elastic',
+  action: {
+    elasticAgent: {},
+  },
+  docsLink: 'test',
+};
+
+describe('KibanaPageTemplate', () => {
+  test('render noDataConfig && solutionNav', () => {
+    const component = shallow(
+      <KibanaPageTemplate noDataConfig={noDataConfig} solutionNav={solutionNav} />
+    );
+    expect(component).toMatchSnapshot();
+  });
+
+  test('render noDataConfig', () => {
+    const component = shallow(<KibanaPageTemplate noDataConfig={noDataConfig} />);
+    expect(component).toMatchSnapshot();
+  });
+
+  test('render solutionNav', () => {
+    const component = shallow(
+      <KibanaPageTemplate
+        pageHeader={{
+          iconType: 'test',
```

Comment:
Should we be adding a `pageHeader` config option in Storybook?

Reply:
Good call. I tried adding `pageHeader` as a prop to the storybook component as such:
```
const pageHeader = {
  iconType: 'logoKibana',
  title: 'Kibana',
  description: 'Welcome to Kibana!',
  rightSideItems: [<EuiButton fill>Add something</EuiButton>, <EuiButton>Do something</EuiButton>],
};
```

and it doesn't render properly:
<img width="1325" alt="Screenshot 2022-04-06 at 11 04 32" src="https://user-images.githubusercontent.com/1937956/161938715-f000944e-30e3-4b6f-83b9-286473f9951e.png">

Looks like the icon is missing and the title is not the title :/
Inspecting the html, there is no reference to the icon or that should even be there. I verified that the `pageHeader` is correctly passed to `EuiTemplate`. Any idea what might have gone wrong?